generators/app/templates/src/containers/Profile/components/Image/image.component.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import data from "@solid/query-ldflex";
+import { Uploader, useLiveUpdate } from '@inrupt/solid-react-components';
+import { ImageProfile } from '@components';
+
+type Props = {
+  webId: String,
+  toastManager: String
+};
+
+
+export const Image = ({ webId, toastManager, defaultProfilePhoto }: Props) => {
+  const [image, setImage] = useState('');
+
+  const latestUpdate = useLiveUpdate();
+
+  useEffect( () => {
+    fetchPhoto();
+  }, [webId, latestUpdate]);
+  /**
+   * Fetch profile photo from card
+   */
+  const fetchPhoto = async () => {
+    try {
+      if (webId) {
+        // We are fetching profile card document
+        const user = data[webId];
```

Comment:
Will `webId` always be the current user?

If yes, then why not `data.user` (and not pass in `webId`)?

If no, then perhaps calling it `user` is confusing.

Reply:
Yes it's a user I updated to data.user thank you !!!